src/main/webapp/modal-activator.js

```diff
@@ -1,3 +1,10 @@
-function trigger-iFrame() {
+function setupModalButtons() {
+  const modalTriggers = document.querySelectorAll('[data-modal-target]');
+  const modalClosers = document.querySelectorAll('[data-modal-close-button]');
   
+}
+
+function triggerModal(modal) {
+  overlay.classList.add('active');
+  modal.classList.add('active');
```

Comment:
As claudia mentioned, comments might be helpful

Reply:
Addressed fxn headers: d7665f2